doc/extra-large-conference/prosody.cfg.lua.visitor.template

```diff
@@ -0,0 +1,124 @@
+---------- Server-wide settings ----------
+component_interface = '0.0.0.0';
+
+s2s_ports = { 52691 };
+c2s_ports = { 52221 }
+http_ports = { 52801 }
+https_ports = { 52811 }
+
+daemonize = true;
+
+-- we use a common jid for jicofo
+admins = {
+    'focus@auth.meet.jitsi'
+}
+
+-- Enable use of native prosody 0.11 support for epoll over select
+network_backend = 'epoll';
+network_settings = {
+  tcp_backlog = 511;
+}
+
+modules_enabled = {
+      'saslauth';
+      'tls';
+      'disco';
+      'posix';
+
+      'secure_interfaces';
+
+    -- jitsi
+      'websocket';
+      'bosh';
+      's2s_bidi';
+      's2s_whitelist';
+};
+
+s2s_whitelist = {};
+
+external_service_secret = '__turnSecret__';
+
+external_services = {
+    { type = 'stun', host = 'jitmeet.example.com', port = 3478 },
+    { type = 'turn', host = 'jitmeet.example.com', port = 3478, transport = 'udp', secret = true, ttl = 86400, algorithm = 'turn' },
+    { type = 'turns', host = 'jitmeet.example.com', port = 5349, transport = 'tcp', secret = true, ttl = 86400, algorithm = 'turn' }
+};
+
+muc_mapper_domain_base = 'v1.meet.jitsi';
```

Comment:
since this is a template, should this be vX or something?

Reply:
I can change that.